feat: explain polkadotjs (#239)
KILTprotocol/docs: docs/develop/03_workshop/04_attester/01_account.md

To generate an account, we use the `addFromMnemonic()` function on the [`KiltKeyringPair`](https://kiltprotocol.github.io/sdk-js/interfaces/_kiltprotocol_types.KiltKeyringPair.html) generated via the SDK.
The mnemonic is generated with the polkadot function `mnemonicGenerate()`, which generates a 12-word mnemonic.

:::info polkadot.js

The KILT SDK is built on top of the [polkadot.js](https://polkadot.js.org/) library therefore you will find various uses of it in this workshop.

> It provides a set of tools to interact with the KILT blockchain and other substrate based blockchains, provides cryptographic primitives and a serialization framework.

Reviewer: It was a bit hard to follow. 

My proposal:

```suggestion
It provides a set of tools to interact with the KILT blockchain and other substrate based blockchains
In addition the polkadot-js library offers cryptographic primitives and a serialization framework to decode data received from the blockchain.
```

ChatGPTs proposal:
```suggestion
This JavaScript library includes a collection of tools for interacting with the KILT blockchain and other Substrate-based blockchains. Moreover, it offers cryptographic primitives and a serialization framework to facilitate blockchain development.
```

Reviewer 2: One-sentence-one-line tho:

```suggestion
This JavaScript library includes a collection of tools for interacting with the KILT blockchain and other Substrate-based blockchains.
Moreover, it offers cryptographic primitives and a serialization framework to facilitate blockchain development.
```

> We recommend that you familiarize yourself with the [API documentation](https://polkadot.js.org/docs/) to learn more about the functions available and get a deeper understanding of how things work.

Reviewer 2: I would assume the intent is always to understand better how things work, so I would remove that section.

```suggestion
We recommend developers to familiarize themselves with their [API documentation](https://polkadot.js.org/docs/) to learn more about the functions available.
```
